app/src/modules/settings/routes/data-model/field-detail/shared/extension-options.vue

```diff
@@ -70,7 +70,10 @@ export default defineComponent({
 
 		const { collection, field } = storeToRefs(fieldDetailStore);
 
-		const extensionInfo = useExtension(props.type, props.extension);
+		const extensionInfo = useExtension(
+			props.type,
+			computed(() => props.extension)
```

Comment:
Also lets use Vue's `toRefs` to make this reactive:

```js
const { extension } = toRefs(props):
```

Reply:
Done 👍